src/routes/adminRoutes.ts

```diff
@@ -26,6 +27,15 @@ export class AdminRoutes extends BaseRoutes {
     this.router.route("/mess1/all/:id").get(isAdminOrUser, (req: Authenticated, res) => this.getUsersForGroundMess(req, res));
     this.router.route("/mess2/all/:id").get(isAdminOrUser, (req: Authenticated, res) => this.getUsersForUpperMess(req, res));
     this.router.route("/god/:id").post((req, res) => this.dropCollection(req, res));
+
+    //Clean Database every Sunday
+    cron.schedule('* * * * Sunday', () => {
```

Comment:
@debck as the server is hosted on some foreign country so time won't be scheduled as per our IST. So we need to convert it according to IST i.e we can use in UTC format.

So change the code to this after importing **moment**

```const presentDay = moment().utcOffset("+05:30").format("dddd");
    if (presentDay === "Monday") {
      cron.schedule("0 0 0 * * Monday", () => {
        console.log("running a task every Monday");
        Coupon.find({}).then((coupon: any) => {
          if (coupon.length !== 0) {
            Coupon.collection.drop();
            console.log("Database Deleted");
          }
          else {
            console.log("Already deleted database")
          }
        });
      });
    }```